standard-app/templates/configs/_externalsecret.tpl

```diff
@@ -0,0 +1,67 @@
+{{- define "standard-app.externalSecret" -}}
+{{- $name := .name }}
+{{- $userLabels := .labels | default dict }}
+{{- $globalLabels := $.Values.labels | default dict }}
+{{- $defaultLabels := dict "product" $.Release.Name }}
+{{- $labels := merge $defaultLabels $globalLabels $userLabels }}
+{{- $secrets := .secrets }}
+{{- $type := .type }}
+{{- $storeName := .storeName }}
+{{- $secretPath := .secretPath }}
+{{- $refreshInterval := .refreshInterval | default "1m" }}
+
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: {{ $name }}
+  labels:
+    {{- toYaml $labels | nindent 4 }}
+spec:
+  refreshInterval: {{ $refreshInterval }}
+  secretStoreRef:
+    kind: ClusterSecretStore
+    name: {{ $storeName }}
+  target:
+    name: {{ $name }}
+    creationPolicy: Owner
+  {{- if and (kindIs "map" $secrets) (hasKey $secrets "dataFrom") }}
+  dataFrom:
+    {{- range index $secrets "dataFrom" }}
+    - extract:
+        key: {{ . }}
+    {{- end }}
+  {{- else }}
+  data:
+    {{- if eq $type "azure" }}
+      {{- range $key, $value := $secrets }}
+    - secretKey: {{ $key }}
+      remoteRef:
+        key: {{ $value }}
+      {{- end }}
+    {{- end }}
+    {{- range $secret := $secrets }}
+      {{- if kindIs "map" $secret }}
+    - secretKey: {{ $secret.secretKey }}
+      remoteRef:
+        {{- if eq $type "gcp" }}
+        key: {{ printf "%s_%s" ($.Release.Name | upper) $secret.secretKey }}
+        property: {{ ( $secret.property | default $secret.secretKey ) }}
```

Comment:
👋🏼 @kirill-cloudkite - I noticed that most GCP secrets are stored as separate secret entries rather than as properties within a single secret. So maybe enforcing a property field might be too strict.
Would it make sense to do something like this instead?
```
property: {{ $secret.property | default "" }}
```
That way, if a secret includes a property, it gets used, and if not, it’s still valid. What do you think?